cmd/norduser/main.go

```diff
@@ -134,17 +124,10 @@ func shouldEnableFileshare(uid uint32) (bool, error) {
 	return meshStatus.GetUid() == uid && meshStatus.GetValue(), nil
 }
 
-func startSnap() {
-	usr, err := user.Current()
-	if err != nil {
-		log.Println("Unable to retrieve current user:", err)
-		os.Exit(int(childprocess.CodeFailedToEnable))
-	}
-
+func setupLog() {
 	homeDir, err := os.UserHomeDir()
 	if err != nil {
-		log.Println("Unable to retrieve user home directory:", err)
-		os.Exit(int(childprocess.CodeFailedToEnable))
+		return
```

Comment:
The application is crashing, in snap, if the logger is not configured and there are `fmt.Println` with Broken pipe error.

Reply:
Solved by configuring log output to `stdout` by default(not resolved by git because it's a different line).